tests/sentry/digests/backends/test_redis.py

```diff
@@ -107,3 +107,14 @@ def test_missing_record_contents(self):
         # contents were merged back into the digest.
         with backend.digest('timeline', 0) as records:
             assert set(records) == set([record_2])
+
+    def test_large_digest(self):
```

Comment:
I also added this test to prove the fix but I'm not really sure it's worth it — it's pretty slow and is dependent on the compile flags of the Lua interpreter